### PR TITLE
212 testcase for every endpoint 1

### LINKED ISSUE #212: NGSI-LD Testcase for every endpoint.
A testcase for each endpoint should be created. The main focus will be on subscriptions and batch operations.

### PR DESCRIPTION
**Pull request by djs0109**
closes #212

### CODE REVIEW

File: filip/models/ngsi_ld/context.py [hunk 1]
```diff
@@ -35,14 +37,60 @@ class ContextProperty(BaseModel):
         >>> attr = ContextProperty(**data)
```
Comment (djs0109) [inline filip/models/ngsi_ld/context.py:37]: @LaraRoth seems that there is some unsolved conflict here

**Comment by iripiri** [comment]
### Endpoints (implemented & tested):
Entities

- [x] GET /entities
- [x] POST /entities
- [x] GET /entities/{entityId}
- [x] DELETE /entities/{entityId}
- [x] POST /entities/{entityId}/attrs
- [ ] PATCH /entities/{entityId}/attrs
- [x] PATCH /entities/{entityId}/attrs/{attrId}
- [x] DELETE /entities/{entityId}/attrs/{attrId}

Batch operations
- [x] POST /entityOperations/create
- [x] POST /entityOperations/update
- [x] POST /entityOperations/upsert
- [x] POST /entityOperations/delete

Subscriptions
- [x] GET /subscriptions
- [ ] POST /subscriptions
- [ ] GET /subscriptions/{subscriptionId}
- [ ] PATCH /subscriptions/{subscriptionId}
- [ ] DELETE /subscriptions/{subscriptionId}

**Comment by djs0109** [comment]
Complete unit tests for client endpoints

Entities

- [ ] GET /entities (all parameters are missing, among all `idPattern`, `type`, `attrs`, `options` should be tested.) 
As for `options`
- `sysAttrs`, a representation of NGSI-LD Elements shall be provided so that the system-generated attributes createdAt, modifiedAt are included in the response payload body.
- `keyValues`, a simplified representation of entities shall be provided
- [x] POST /entities
- [ ] GET /entities/{entityId} (parameters `type` and `optinos` are missing)
As for `options`,
- `sysAttrs`, a representation of NGSI-LD Elements shall be provided so that the system-generated attributes createdAt, modifiedAt are included in the response payload body.
- `keyValues`, a simplified representation of entities shall be provided
- [x] DELETE /entities/{entityId}
- [x] POST /entities/{entityId}/attrs
- [ ] PATCH /entities/{entityId}/attrs
- [x] PATCH /entities/{entityId}/attrs/{attrId}
- [x] DELETE /entities/{entityId}/attrs/{attrId}

File: filip/models/ngsi_ld/context.py (hunk 1 above)
Comment (iripiri) [reply filip/models/ngsi_ld/context.py:38]: I  believe the ContextProperty Model is missing the parameters 'createdAt' and 'modifiedAt' which will be sent back by the context broker when the option 'sysAttrs' is used for getting an entity from the broker.

**Pull request merged**